README.md

```diff
@@ -1,2 +1,49 @@
-# rfcs
-RFCs for XState and Stately tools
+# RFCs for XState and Stately tools
+
+This [RFCs](https://en.wikipedia.org/wiki/Request_for_Comments) repository is a place to discuss changes to XState and Stately tools. A Request for Comments (RFC) is a design proposal formatted as a pull request (PR) proposal, including its design’s motivation, technical background, implementation, alternatives and drawbacks.
+
+A considerable part of the value of an RFC is defining the problem clearly, collecting use cases, showing how others have solved a problem, etc. Coming up with a design is very iterative and only one part of the process. An RFC can provide tremendous value without the team accepting the design as described.
+
+Minor changes may not need to go through the RFC process and can rely on issues and pull requests. Major changes should always go through the RFC process, where ‘major’ implies significant changes either to public interfaces or internal implementation details, particularly those that could be controversial or involve breaking changes.
+
+Anyone can submit RFCs. The most common reasons the team rejects an RFC is that it has significant design gaps or flaws, does not work cohesively with all the other features, or does not fall into our view of the scope of XState or the Stately tools. 
+
+However, getting merged is not the only success criteria for an RFC. Even when the design does not match the direction we’d like to take, we find RFC discussions very valuable for research and inspiration. Whenever we start work on a related area, we check the RFCs in that area and review the use cases and concerns the community members have posted. When you send an RFC, your primary goal should not necessarily be to get it merged into XState or the Stately tools as is, but to generate a valuable discussion with the community members. If your proposal later becomes accepted, that’s great. But even if it doesn’t, your work isn’t for nothing! The resulting discussion often informs the next proposal in the same problem space, whether the proposal comes from the community or the team.
+
+Our RFC process is inspired by the process adopted by [Svelte](https://github.com/sveltejs/rfcs), [React](https://github.com/reactjs/rfcs), [Ember](https://github.com/emberjs/rfcs) and others. The process itself is subject to change, or may even be abandoned, as we gain experience.
+
+
+## The process
+
+![Statechart representing the RFC process detailed below.](https://user-images.githubusercontent.com/266663/169092679-3f64947d-31e6-4e04-bdba-baf408dfd832.png) [View this statechart in the Stately editor](https://stately.ai/registry/editor/share/eee673df-9f4d-4965-beb8-1a649676521a)
```

Comment:
nice Studio flex 💪 :P 

soon we should also be able to include a light-mode picture like described here: https://github.blog/changelog/2021-11-24-specify-theme-context-for-images-in-markdown/